style.css

```diff
@@ -0,0 +1,943 @@
+html,
+body {
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+li {
+  list-style-type: none;
+}
+
+.descrp21 {
+  background-color: #e7e7e7;
+  margin-top: -35px;
+  padding-top: 20px;
+}
+
+.index-header img {
+  margin-top: 20px;
+  margin-left: 20px;
+}
+
+.backimg {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  display: block;
+  overflow: hidden;
+  z-index: -1;
+  height: 51%;
+}
+
+.summit {
+  width: 100%;
+  background: rgba(255, 255, 255, 0.829);
+  background-size: cover;
+}
+
+.title-h1 {
+  color: #ec5242;
+  font-family: "Poppins", sans-serif;
```

Comment:
- [ ] [OPTIONAL] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.font` class in multiple places or simply put the `font-family` property in the `body` selector and all other children elements going to inherit it by default.

- (your code)

```css
.title-h1 {
  /* ...code */
  font-family: 'Poppins', sans-serif;
  /* ...code */
}

.abouttitle-h1 {
  /* ...code */
  font-family: 'Poppins', sans-serif;
  /* ...code */
}
```

- suggestion

```css
.font {
  font-family: 'Poppins', sans-serif;
}
```

> **Note**: You can also use this technique with multiple properties like: `display: flex`.

Reply:
Done